imports/plugins/custom/footer/client/components/customCoreLayout.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import { Components, replaceComponent } from "@reactioncommerce/reaction-components";
+import Footer from "./footer";
+import Blaze from "meteor/gadicc:blaze-react-component";
+import { Template } from "meteor/templating";
+
+const CustomCoreLayout = ({ actionViewIsOpen, structure }) => {
+  const { layoutFooter, template } = structure || {};
+
+  const pageClassName = classnames({
+    "page": true,
+    "show-settings": actionViewIsOpen
+  });
+
+  return (
+    <div className={pageClassName} id="reactionAppContainer">
+      <Components.NavBar />
+
+      <Blaze template="cartDrawer" className="reaction-cart-drawer" />
+
+      { Template[template] &&
+        <main>
+          <Blaze template={template} />
+        </main>
+      }
+
+      { Template[layoutFooter] &&
+        <Blaze template={layoutFooter} className="reaction-navigation-footer footer-default" />
+      }
```

Comment:
This file replaces the entire coreLayout in order to add a footer. I suggest replacing the layoutFooter template instead

Reply:
All feedback implemented. Thanks for pointing this out.